doc/source/development/rfc/rfc98_build_requirements_gdal_3_9.rst

```diff
@@ -0,0 +1,165 @@
+- libpng >= 1.6.0: removes outdated code paths in the PNG driver
+- libsqlite3 >= 3.31: removes outdated code paths in the SQLite and GPKG drivers
+- libopenjp2 >= 2.3.1: removes outdated code paths in the OpenJPEG driver
+- libnetcdf >= 4.7 built with NC4 support enabled (i.e. libnetcdf built against
+  libhdf5): removes #ifdef code paths in netCDF driver. The netCDF multidimensional
+  code already requires NC4.
+- libhdf5 >= 1.10: removes outdated code paths in the HDF5 driver
+
+C++17 capable compilers
+-----------------------
+
+From https://en.wikipedia.org/wiki/C%2B%2B17, compilers supporting C++17 are:
+
+- GCC >= 8
+- clang >= 5
+- Visual Studio >= 2017 15.8 (MSVC 19.15)
+
+C++14 and C++17 features
+------------------------
+
+Features that can be used in the code base (not an exhaustive list):
+
+- Use of ``std::make_unique<>``  instead of ``cpl::make_unique<>``
+- Use of ``[[fallthrough]]`` instead of ``CPL_FALLTHROUGH``
+- Use of ``[[maybe_unused]]`` instead of ``CPL_UNUSED``
+- Nicer iteration in ``std::map`` with ``for (const auto &[key, value]: my_map )``
+  (more generally "structured binding declarations")
+
+Banned features:
+
+- Use of ``std::filesystem`` (https://en.cppreference.com/w/cpp/filesystem)
+  is not appropriate since our existing :cpp:class:`VSIFilesystemHandler`
+  mechanism has broader support for all our /vsi specific file systems.
+
+Changes in continuous integration
+---------------------------------
+
+Continuous integration is modified to test configurations that have at least
+the new set of build requirements.
+
+Changes in SWIG bindings
+------------------------
+
+None
+
+Backward compatibility
+----------------------
+
+No change in API and ABI
+
+Documentation
+-------------
+
+The Build requirements documentation page will be updated.
+
+Testing
+-------
+
+The existing autotest suite should continue to pass.
+
+Related tickets / pull requests
+-------------------------------
+
+- https://github.com/OSGeo/gdal/pull/8680: CI: Remove Ubuntu 18.04 configurations
+- https://github.com/OSGeo/gdal/issues/8270: Bump to C++17
+- https://github.com/OSGeo/gdal/pull/8687: Switch default C++ standard to C++17
+- https://github.com/OSGeo/gdal/pull/8710: Cxx17 tunings
+- https://github.com/OSGeo/gdal/pull/8716: C++17: use if constexpr() construct
+- https://github.com/OSGeo/gdal/pull/8723: C++17: replace CPL_FALLTHROUGH by standard [[fallthrough]];
+- https://github.com/OSGeo/gdal/pull/8725: C++17: use structured bindings declaration and class template argument deduction for std::pair()
+- https://github.com/OSGeo/gdal/issues/8751: Bumping minimum CMake version to 3.16
+- https://github.com/OSGeo/gdal/issues/8796: Add CI test we can use GDAL public/installed headers with C++11
+- https://github.com/OSGeo/gdal/pull/8804: Make build compatible of -DCMAKE_UNITY_BUILD=ON for faster builds
+
+Related RFCs
+------------
+
+- This RFC supersedes :ref:`rfc-68`
```

Comment:
Will this link work once it's published? It doesn't seem to now.

Reply:
it's there: https://gdal.org/development/rfc/rfc68_cplusplus11.html . The link will become active once this is merged and fully rendered